### PR TITLE
Add 'connection established' log info msg to connection.py

### PR DESCRIPTION
**Pull request by david9600**
Add an info log message to indicate successful connection to the panel. This would be useful when investigating a config or network issue, since it gives immediate feedback, particularly now that the HA system log auto-updates.

### CODE REVIEW

File: bosch_alarm_mode2/connection.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def set_max_commands_in_flight(self, command_count):
         self._command_semaphore = asyncio.Semaphore(command_count)
 
     def connection_made(self, transport):
+        LOG.info("Panel connected.")
```
Comment (mag1024) [inline bosch_alarm_mode2/connection.py:29]: By analogy with line 32: "Connection established."

Comment (david9600) [reply bosch_alarm_mode2/connection.py:29]: Fine by me ... do I need to edit the PR? (which I'm not sure how to do, other than submitting a new one!)

Comment (sanjay900) [reply bosch_alarm_mode2/connection.py:29]: Push another commit to your branch, and it will then update the PR

Comment (david9600) [reply bosch_alarm_mode2/connection.py]: OK... I think I've managed to do that (using GitHub web interface).

**Pull request merged**